src/custom/pages/About/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import useMarkdown from '../../hooks/useMarkdown'
+import { Title, Content, AppBodyMod } from './About.styled'
+import markdownContent from './about.md'
+import ReactMarkdown from 'react-markdown'
+
+export default function About() {
+  return (
+    <AppBodyMod>
+      <Title>About</Title>
+      <Content>
+        <ReactMarkdown>{useMarkdown(markdownContent)}</ReactMarkdown>
```

Comment:
Ideally, we would remove any direct depedency with an external library like this. 

One suggestion is, make a Markdown component, that just reexport ReactMarkdown, so no actual implementation

Reply:
Yeah my thinking was to cater the Hook to do that part, so it becomes more of a Markdown specific Hook. Able to help out there?

Reply:
> One suggestion is, make a Markdown component, that just reexport ReactMarkdown, so no actual implementation
That would still depend on ReactMarkdown in the end?

Reply:
yes, but only in a single point in the app, rather than in every place you use it. 
however, if you implement a markdown component and thats the only one using it, you would be right that its not necesary

Reply:
Gotcha, so you'd want a general purpose simple component which wraps the ReactMarkdown as a (for example) <Content></Content> ?

Reply:
If so, let's do in a separate PR?

Reply:
yes, separate. Actually, when we do the policies is a good moment, it would be when we start to use the library a second time, so is a good moment to refactor. This PR is perfect as it is